### DISCUSSION TITLE
Obtaining the number of links between Peaks and Genes

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#658] [Questions / Documentation] [answered]
Is there a way of determining the number of peak2gene links **for each gene** that meet the cutoffs?

**Comment by rcorces** [comment, marked as answer]
This information should be available from the return of `getPeak2GeneLinks()` when `returnLoops = FALSE` via the `idxRNA` column. This is explained briefly in this section of the manual:
https://www.archrproject.com/bookdown/peak2genelinkage-with-archr.html

**Comment by sylestiel** [comment]
Can you elaborate some more. I tried writing  the peakset and geneset files  under peak2gene links to csv but although I see the gene names with IRanges in the later and GRanges in the former I still don't understand how to associate the **actual number** of Links associated with a given gene(s).

**Comment by sylestiel** [comment]
If idx is the index for each message then the only way of arriving at the number of links seems rather laborious and manually. If there is a simpler way of doing so do advice.

**Reply by rcorces** [reply]
From the manual:
"the indexes for the scRNA-seq gene are stored in the idxRNA column"
so `idxRNA` is a numeric index of the gene from your RNA data. so if you count the number of observations of each `idxRNA` and map them back to the gene symbol, you get what you are looking for.

**Comment by sylestiel** [comment]
Thanks a bunch! Already on it. Hence the manual part.
